internal/envcheck/tasks.go

```diff
@@ -0,0 +1,74 @@
+package envcheck
+
+import (
+	"errors"
+	"os"
+	"os/exec"
+	"strings"
+
+	"golang.org/x/sys/execabs"
+)
+
+type task struct {
+	name, hint string
+	test       func() (string, error)
+}
+
+var tasks = []*task{
+	{"Go compiler", "Checking Go is installed and up to date", func() (string, error) {
+		path, err := exec.LookPath("go")
```

Comment:
I think you might want to use `execabs` here too. This applies to more parts of the PR.

Reply:
Thanks, fixed